### PR TITLE
Create Fermions_in_double_well_qiskit.ipynb

### PR DESCRIPTION
**Pull request by fretchen**
Attached you can find one tutorial on experiments in the Jochim group that I prepared for teaching. I am not sure how it fits into the series of tutorials, but it might be nicely integrated ...

### CODE REVIEW

**Review by eggerdj — CHANGES_REQUESTED** [review]
Overall this is looking good but we need more description of what is going on. The biggest change needed is more description to guide the user through the tutorial. Here are some additional changes we should implement:
* Add title of the paper in the introduction text. This text also has some typos.
* Feels unnecessary to have black formatting on the data.
* Add a description of the first plot. What are we seeing?
* scipy import is randomly placed in the NB.
* Analytical prediction: write out what $|L\rangle$ and R stand for. State what this section is doing. E.g. "Here we numerically solve the dynamics of the system by exponentiating the Hamiltonian.". Perhaps we can package this calculation into a small function so that you can do:
```
ax1.plot(t_analytical * 1e3, solve_dynamics(U=0), "r-", label="U = 0", linewidth=4, alpha=0.5)
ax2.plot(t_analytical * 1e3, solve_dynamics(U=J), "b-", label="U = J", linewidth=4, alpha=0.5)
```
* Are the labels correct since you write `U = 0.7 * J;` in one of the cells?
* Describe the plot in the analytic section.
* After the analytic section we start building circuit. We should state that we are doing this and why. E.g. "We now show that the experiment presented above can be described in a circuit language using Qiskit cold atom. To do this we will need a quantum circuit with two registers of four modes each to describe ... . We first must load the fermions into the tweezers, see tutorial XX for details. The fully dynamics is contained in one gate, i.e. the `FermiHubbard` gate, that encapsulates the system's dynamics."
* The plot in output 18 needs a description and labels. E.g. we see hopping, the simulation and the data from Selim match, etc.
* Language: `As you can see the back-end is only implementation a limited set of back-ends such that the full circuit has to be trotterized.`
* Add comment describing the variables, e.g. 
```
# Variable description
J = ...

# Variable description
U = 0.7 * J
```

**Comment by LaurinFischer** [comment]
I have now reworked the Qiskit part of the notebook to make full use of the functionalities provided by the package.

File: docs/tutorials/08_fermions_in_double_well.ipynb [hunk 1]
```diff
@@ -0,0 +1,891 @@
...
+    "ax2.legend()\n",
+    "ax1.grid()\n",
+    "ax2.grid()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "These dynamics have an intuitive explanation: Without interaction, both atoms tunnel back and forth between $|L\\rangle$ and $|R\\langle$ at the same rate, which leads to sinusoidal interactions. With interaction, the atoms repel each other (as $U>0$) whichs forms a complicated beating pattern of the tunneling. "
```
Comment (eggerdj) [inline docs/tutorials/08_fermions_in_double_well.ipynb:254]:
```suggestion
    "These dynamics have an intuitive explanation: Without interaction, both atoms tunnel back and forth between $|L\\rangle$ and $|R\\rangle$ at the same rate, which leads to sinusoidal interactions. With interaction, the atoms repel each other (as $U>0$) whichs forms a complicated beating pattern of the tunneling. "
```

File: docs/tutorials/08_fermions_in_double_well.ipynb [hunk 2]
```diff
@@ -0,0 +1,891 @@
...
+    "qc.measure_all()\n",
+    "\n",
+    "qc.draw(output=\"mpl\", scale=0.8)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To directly simulate the experiment, we can define the measured observable and the run a list of circuits with different evolution times on the simulator backend. "
```
Comment (eggerdj) [inline docs/tutorials/08_fermions_in_double_well.ipynb:322]:
```suggestion
    "To directly simulate the experiment, we can define the measured observable and then run a list of circuits with different evolution times on the simulator backend. "
```

File: docs/tutorials/08_fermions_in_double_well.ipynb [hunk 3]
```diff
@@ -0,0 +1,891 @@
...
+    "circuits_int = [qc.bind_parameters({hop_param: J*t, int_param: U*t}) for t in times]\n",
+    "# measure the observable from simulated shots\n",
+    "means_int = backend.measure_observable_expectation(circuits_int, observable=NR, shots=1000)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's compare the qiskit's simulations to the experimental data and the numerical model from above:"
```
Comment (eggerdj) [inline docs/tutorials/08_fermions_in_double_well.ipynb:366]:
```suggestion
    "Let's compare the Qiskit simulations to the experimental data and the numerical model from above:"
```

**Review by eggerdj — CHANGES_REQUESTED** [review]
Looks awesome. A few minor typos.

File: docs/tutorials/08_fermions_in_double_well.ipynb [hunk 4]
```diff
@@ -0,0 +1,891 @@
...
...AADAn8QeYL1YuXJlenp6MmbMmCRJW1tbli5dWuFUANWl5XtHp199fTZu+nBneZRkt5/f/+CjnHx8yx7Lr66urlxyySVpbW0tWfmVJIMHDczxE76318wbN32Ybzce0qfyK9nx6GZbW1uefPLJ/Z4ZAAD40ynAejFs2LCsWbMmGzZsSJI899xzGTt2bIVTAVSXfv3qc+HZ09O/f789nnfK94/L8Kahezxn1qxZOeigg/LAAw/sz4i9ah17TMaP/u4ez2k46Fs597ST9njO5s2bs3HjxiQ79gB7+umn09zcvN9yAgAA+489wHrR1NSUe+65J6eddloGDBiQww8/PHPnzq10LICqM/Swg3PGtBPyL8/3fpfssKbDM/3EiXu8xquvvpq5c+dm3LhxmThxx7lXXnllrr/++v2eN0nq6urywz8/OX/Y8H42b9na6zk/OnNaDhg8aI/X2bx5c84999x0dHSku7s7J5xwgv8xGAAAqpQ9wHrZA2xPenp6/C9fAP9Pd09PfvZPi/LOHzbscnxA/365/vLzc/hhB1cm2F6s+f26/OyJf9vt+El/Ni7nzPh+BRIBAACl4hHIfdT+q9/mHxY+l+1dXZWOAlAV6uvqcsGZ0zJ40MBdjp85fXLVll9JMvroozJ54q6Ptw897ODMnHJ8hRIBAAClUvMF2KpVq3L++eensbExgwcPzqhRo3L77beX5Ls6Ojrz8q9+m84vtqd/vz3veQPwTdJw0Ldy3uk/2Pn62O8O261cqkZnTjshjYc0JEnq6+ty0dmnZMAAuwMAAEDR1HQBtnz58kyePDmrV6/O/fffn0WLFuXWW2/Ne++9V5LvW7pidbZu68ipJ00qyfUBatmEscdkwpijc8DgQbngjKk18bj4wIEDcuFZ01JXV5dTT5qUI49orHQkAACgBGp6D7Bp06blzTffzDvvvJOGhoZ9+uxRI0cnSS758c2liAYAAABACd3zt7P6fG7N3gG2devWLFmyJBdffPE+l18AAAAAfHPU7EYnH3/8cbq7uzNs2LCv9fmRRzUl6Vtb2NHRmXsfmp/h3xmaK350xtf6PgAAAAAqo2YLsEMOOST19fXZsGHD1/r8lu07NrG/9d6H+/yZNf/x7j6dDwAAAEBpfCMegTzwwAMzZcqUzJ8/P5999tk+f/7Ucy7KqedcVIJkAAAAAFSTmt4Ef/ny5Zky
```
Comment (eggerdj) [inline docs/tutorials/08_fermions_in_double_well.ipynb:757]:
```suggestion
    "You will now have to save the job and wait until the job status changes to done. "
```

Comment (LaurinFischer) [reply docs/tutorials/08_fermions_in_double_well.ipynb]: addressed in fd7f90d

**Review by eggerdj — APPROVED** [review]
LGTM!

**Pull request merged**